libcore-stub/src/main/java/libcore/java/security/StandardNames.java

```diff
@@ -520,6 +520,10 @@ private static void provideSslContextEnabledProtocols(
             provide("Cipher", "DESEDE/OFB/PKCS7PADDING");
 
             // Provided by our OpenSSL provider
+            provide("AlgorithmParameters", "ChaCha20");
```

Comment:
@flooey IIRC, there was quite a bit of hackery to this and other files to make it work for openjdk.  I suspect there will be some work to merge them with the Android version. 

No action here, just an FYI.

Reply:
Yeah, I didn't merge in everything from Android, just the bits that were obviously needed.  I expect it will take a bit of effort to make it work.